tests/performance-test/performance-test.sh

```diff
@@ -235,20 +224,20 @@ while true; do
             printf "\nQdr edge test pod established\n"
             make_service_monitors
 
-            QDR_READY=$(oc logs $(oc get pod -l application=qdr-test -o jsonpath='{.items[0].metadata.name}') | grep -o "Listening" || true)
+            QDR_READY=$(oc logs $(oc get pod -l application=qdr-test -o jsonpath='{.items[0].metadata.name}') | grep -o "Listening")
 
             if [ -z "$QDR_READY" ]; then 
                 printf "Waiting on router to complete initialization"; ellipse
             else
-                oc logs $(oc get pod -l application=qdr-test -o jsonpath='{.items[0].metadata.name}')
+                #oc logs $(oc get pod -l application=qdr-test -o jsonpath='{.items[0].metadata.name}')
                 STAGE="TARGET"
             fi
         ;;
         "TARGET")
-            TARGETS=$(curl --silent "http://$(oc get route prometheus -o jsonpath='{.spec.host}')/api/v1/targets")
-            QDRWHITE=$(echo "$TARGETS" | grep -o '"__meta_kubernetes_service_name":"qdr-white"' || true)
-            QDRTEST=$(echo "$TARGETS" | grep -o '"__meta_kubernetes_service_name":"qdr-test"' || true)
-            PROM=$(echo "$TARGETS" | grep -o '"__meta_kubernetes_service_name":"prometheus-operated"' || true) 
```

Comment:
You probably should not delete `|| true` here. If the service monitors do not exist yet, grep will exit with a non-zero code causing the whole test to exit. Given the script is supposed to wait until the service monitor is available, that isn't ideal functionality. Unless you tested this on a blank slate and it works as intended, I am fairly certain this is not what you want

Reply:
I took these out while futzing with something else and never really considered why exactly they were there. I'll put them back, you're totally right, makes perfect sense.